### PR TITLE
test(p/uint256): Increase Test Coverage for `uint256` Package

### PR DESCRIPTION
**Pull request by notJoon**
# Description

Screenshot 2024-10-10 at 2 53 28 PM

Increased the test coverage of the `p/demo/uint256` package. Previously, only about 40% was covered, but now it has increased to 90% (checked in go).

The existing implementation of the uint256 function is unmodified except for modifying it to use the strconv package.

- [X] Added new tests, or not needed, or not feasible
- [ ] Provided an example (e.g. screenshot) to aid review or the PR is self-explanatory
- [ ] Updated the official documentation or not needed
- [ ] No breaking changes were made, or a `BREAKING CHANGE: xxx` message was included in the description
- [X] Added references to related issues and PRs
- [ ] Provided any useful hints for running manual tests
- [ ] Added new benchmarks to [generated graphs](https://gnoland.github.io/benchmarks), if any. More info [here](https://github.com/gnolang/gno/blob/master/.benchmarks/README.md).
</details>

### CODE REVIEW

File: examples/gno.land/p/demo/uint256/uint256.gno
Comment (notJoon) [reply examples/gno.land/p/demo/uint256/uint256.gno:146]: This was just my personal preference. should I revert?

Comment (jefft0) [reply examples/gno.land/p/demo/uint256/uint256.gno:146]: I think so. My guess is that core devs will approve the PR more quickly if it just updates the tests. It's up to you.

Comment (notJoon) [reply examples/gno.land/p/demo/uint256/uint256.gno]: good. reverted just in case

**Review by jefft0 — APPROVED** [review]
In examples/gno.land/p/demo/uint256, `gno test .` passes .

**Comment by jefft0** [comment]
Removed the "review/triage-pending" label because jefft0 approved. Ready for core dev review.

**Pull request merged**
